rotkehlchen/assets/asset.py

```diff
@@ -470,7 +470,7 @@ class EvmToken(CryptoAsset):
     chain_id: ChainID = field(init=False)
     token_kind: EvmTokenKind = field(init=False)
     decimals: int = field(init=False)
-    protocol: str = field(init=False)
+    protocol: str | None = field(init=False)
```

Comment:
In the globaldb we use none and in the initialize field we had None as default value but the type didn't allow none. While writting the test this check failed and I have updated it